electrumx/server/block_processor.py

```diff
@@ -2662,6 +2727,7 @@ def advance_txs(
                 # Only parse witness information for Atomicals if activated
                 atomicals_operations_found_at_inputs = parse_protocols_operations_from_witness_array(tx, tx_hash, self.is_density_activated(height))
                 if atomicals_operations_found_at_inputs:
+                    # TODO
```

Comment:
Anything todo here?

Reply:
dft operation

Reply:
Should it be done now? If everyone must resync then we should do it now?